tests/aws/services/dynamodb/test_dynamodb.py

```diff
@@ -238,55 +238,6 @@ def test_list_tags_of_resource(self, aws_client):
 
         aws_client.dynamodb.delete_table(TableName=table_name)
 
-    @markers.aws.only_localstack
```

Comment:
Moving this test to a new `test_dynamodb_streams.py` file (only refactoring, no functional changes)..